local/run.go

```diff
@@ -442,17 +458,29 @@ func (a *Agent) Stop() error {
 }
 
 func createAgentBootstrap(checkoutPath string) (*os.File, error) {
-	tmpFile, err := ioutil.TempFile(os.TempDir(), "bootstrap-")
+	tempFileNamePattern := "bootstrap-"
+	if runtime.GOOS == "windows" {
+		tempFileNamePattern = "bootstrap-*.bat"
```

Comment:
The `.bat` extension is really important, without it the agent won't find the boostrap script.

Reply:
`.cmd` is the more modern choice (unless powershell is an option!): https://serverfault.com/questions/17899/whats-the-difference-between-cmd-and-bat-files

Reply:
Might be an option, have to check the agent source code.